src/helm/benchmark/run_expander.py

```diff
@@ -819,6 +819,57 @@ def expand(self, run_spec: RunSpec) -> List[RunSpec]:
             )
             for value in self.values
         ]
+    
+    
+class ChatMLRunExpander(RunExpander):
+    
+    name = "chatml"
+    
+    def __init__(self):
+        self.name = type(self).name
+
+    def expand(self, run_spec: RunSpec) -> List[RunSpec]:
+        adapter_spec = run_spec.adapter_spec
```

Comment:
Could you give a concrete example of the three cases with an example of the input/output and the constructed prompt?  Otherwise, it's pretty hard to understand.

Reply:
Delete msmarco (it only applies to full_functionality models)

Reply:
Actually, I still think msmarco is something we can support. It is possible to provide logprob for the completion.

Reply:
OK, let's leave this in then.